src/app/components/action-button/index.html

```diff
@@ -72,7 +72,7 @@
       propertyName="route"
       isOptional="true"
     >
-      Specifies an Angular router link for the action button. This property accepts a custom object with a <sky-code>commands</sky-code> property to specify router commands and an <sky-code>extras</sky-code> property to specify <a href="https://angular.io/api/router/NavigationExtras">Angular <sky-code>NavigationExtras</sky-code></a>.
+      Specifies an Angular router link for the action button. This property accepts a custom object with a <sky-code>commands</sky-code> property to specify router commands and an <sky-code>extras</sky-code> property to specify <a href="https://angular.io/api/router/NavigationExtras">Angular <sky-code>NavigationExtras</sky-code></a>. This property passes the <sky-code>commands</sky-code> property to the <a href="https://angular.io/api/router/RouterLink">Angular <sky-code>routerLink</sky-code></a>.
```

Comment:
I think we could tweak this slightly to better clear up any confusion.

Looking at https://github.com/blackbaud/skyux-layout/blob/7529c88cb598eb420cc81e856eeda4a5ac835c24/src/app/public/modules/action-button/action-button.component.html#L1-L12 we can see the behavior when the `route` property is used.

Specifically, we turn around and pass the following properties to Angular:
-  `permalink.route?.extras?.fragment` to [`fragment`](https://angular.io/api/router/NavigationExtras#fragment)
- `permalink.route?.extras?.queryParams` to [`queryParams`](https://angular.io/api/router/NavigationExtras#queryParams)
- `permalink.route?.extras?.queryParamsHandling` to [`queryParamsHandling`](https://angular.io/api/router/NavigationExtras#queryParamsHandling)
 - `permalink.route?.commands` to [`routerLink`](https://angular.io/api/router/RouterLink)

Reply:
I got excited and merged because we Slack chatted about this, sorryyyy! 😳😬  

Would it make sense to add something like, "This property passes the following properties to Angular: " and then include the list above? 

@blackbaud-johnly Thoughts on organization for that?

Reply:
That organization makes sense to me, but do we need "to Angular" in the sentence leading into the list @Blackbaud-BobbyEarl?

Reply:
Probably not necessary as long as we link to the individual parts.  The original confusion around this was that it's in turn handing things over to Angular.